subsys/net/ip/Kconfig

```diff
@@ -426,6 +426,14 @@ config NET_MAX_MCAST_ROUTES
 	  This determines how many entries can be stored in multicast
 	  routing table.
 
+config NET_MCAST_ROUTE_MAX_IFACES
+	int "Max number of network interfaces per multicast routing entry"
+	default 1
```

Comment:
Could add a `range 1 xxx` here, so that it's not possible to misconfigure.

Reply:
As far as I know, there is no way to fetch the number of network interfaces so I have set max to arbitrary chosen 8.